File: Semestres/1.Febrero-Junio 2017/IPOO/Ejemplos/15Marzo/ejemplo1.cpp
/*
 * Autor: Carlos A Delgado
 * Archivo: Ejemplo1.cpp
 * Descripción: Ejemplo retorno void
 * Fecha: 15-Marzo-2017
 */
#include <iostream>

using namespace std;

void funcion(){
	return;
}

bool funcion2(){
	return false;
}

int main(){
	funcion();
	cout << funcion2() << endl;
	return 0;
}

File: Semestres/10.Agosto-Diciembre 2021/FADAUSB/11-Sep/Ejemplo1.py
"""
Autor: Carlos A Delgado
Fecha: 11 de Sep de 2021
"""
import math
def recurrencia(n):
    if n==1:
        return 8
    else:
        return 7*recurrencia(n/6)+n+1

def formula(n):
    lg = math.log(7)/math.log(6)
    p1 = (n**lg -1)/6
    p2 = 6*n*(n**(lg-1)-1)
    p3 = 8*(n**lg)
    return p1+p2+p3

n = [6**x for x in range(0,12)]

for ni in n:
    print("n ",ni," RR ",recurrencia(ni)," Formula ",formula(ni))
